product-service/serverless.ts

```diff
@@ -19,16 +20,50 @@ const serverlessConfiguration: Serverless = {
   provider: {
     name: 'aws',
     runtime: 'nodejs12.x',
-    region: 'eu-west-1',
+    region: config.region,
     apiGateway: {
       minimumCompressionSize: 1024,
     },
     environment: {
-      PG_HOST: 'lesson4-instance.cr99zrmm0khq.us-east-1.rds.amazonaws.com',
+      PG_HOST: `lesson4-instance.cr99zrmm0khq.${config.region}.rds.amazonaws.com`,
       PG_PORT: '5432',
-      PG_DATABASE: 'lesson4',      
+      PG_DATABASE: 'lesson4',   
+     
     }    
   },
+  resources: {
+    Resources: {
+      SQSQueue: {
+        Type: 'AWS::SQS::Queue',
+        Properties: {
+          QueueName: 'catalog-items-queue',
+          ReceiveMessageWaitTimeSeconds: 20,
+        }
+      },
+      SNSTopic: {
+        Type: 'AWS::SNS::Topic',
+        Properties: {
+          TopicName: 'create-product-topic',
+        },
+      },
+      SNSSubscription: {
+        Type: 'AWS::SNS::Subscription',
+        Properties: {
+          Endpoint: 'maximsc1285@gmail.com',
```

Comment:
it would be better to storage such data in env file with usage of serverless-dotenv-plugin